### PR TITLE
Updating create-component

### PR DESCRIPTION
**Pull request by silviopaganini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+```bash
+$ yarn add @groselha/create-component -D
+```
+
+## Usage
+
+Create a component and follow the prompts
+```bash
+create-component <Component Name>
+```
```
Comment (peduarte) [inline README.md:15]: `npx` please 😂

File: template/Component.test.enzyme.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16'
```
Comment (peduarte) [inline template/Component.test.enzyme.js]: missing `;` at the end lol

**Review by peduarte — CHANGES_REQUESTED** [review]

**Comment by peduarte** [comment]
@silviopaganini <3

**Comment by silviopaganini** [comment]
updated the PR, we don't need the Adapter as long as it's on the setupJest.js

**Review by lucasmotta — APPROVED** [review]

**Pull request merged**